lib/send-sms.rb

```diff
@@ -0,0 +1,13 @@
+require("bundler")
```

Comment:
This is a really neat way of nesting an API within a method. I couldn't really understand it, so I've just used the standard twilio ruby file. That's great that you've linked it so well with your customer.rb